app/commands.py

```diff
@@ -612,11 +637,56 @@ async def requests(ctx: Context) -> Optional[str]:
             l.append(f"Failed to find requested map ({map_id})?")
             continue
 
-        l.append(f"[{player.embed} @ {dt:%b %d %I:%M%p}] {bmap.embed}.")
+        l.append(
+            f"#{id}: {bmap.embed}\n({dt:%b %d %I:%M%p}) Requested for "
+            + f"{requested_status} by {p.embed}. "
+            + (f"Additional comment: {comment}" if comment else ""),
+        )
 
     return "\n".join(l)
 
 
+@command(Privileges.NOMINATOR, aliases=["dr", "decline"], hidden=True)
+async def declinerequest(ctx: Context) -> Optional[str]:
```

Comment:
i'm unsure about this function as a whole regarding the fact it clears **all** requests. there could be a ranked request that you want to deny but not deny the loved one. perhaps declining by request ID should be an enforced behaviour? curious to hear other's thoughts on this